Week 1 Quiz Answers

```diff
@@ -0,0 +1,112 @@
+
+Question #5
+
+What does the following selector do? ul.dropdown > li?
+
+Select 1:
+
+[X] Selects all li's which are directly inside a ul of class dropdown (children)
+[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[] Selects all ul's of class dropdown, only if their children are exclusively li's
+Question #6
+
+Identify the three places CSS can go, and rank them in terms of specificity:
+
+1. linked stylesheet
+2. Embed CSS on tags themselves
+3. Create a CSS block in web page (typically between <head> tags)
+
+
+A stylesheet is recommended, for it is clean, and makes reading HTML much easier.
+Embedding CSS on to the tags themselves is specific regarding that particular tag,
+but it will make the HTML look quite messy. Also, it may become repetitive
+should a developer want to style a container, but as well as any elements within
+that container.
+Creating a CSS block within the web page is very broad.
+
+
+Git
+
+Question #7
+
+Which of the following represents a correct workflow for submitting a PR on a non-master branch? (ignore the lack of commit messages)
+
+Select 1:
+
+X fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+[] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+[] git clone <ga_dc_url>; git branch <charlie_solution>; git add <files>; git commit; git push; create pull request
+[] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git pull; create pull request
+
+Question 8
+
+What is the difference between a fork and a clone?
+//A clone is a copy of a remote repository that will live as a local repository. If any changes are made to the clone,
+no one will see them until they are pushed up to the original repository. Forking a repository allows a
+developer to make changes without affecting the original source code. It makes a local copy in one's own git repository
+account. When changes are made, that developer may send a 'pull request' to ask if the original developer will accept
+the code.
+
+Your answer here.
+Question 9
+
+How is git pull related to git fetch?
+
+//'git fetch' brings one's local copy of a remote repository up to date.
+This command is important because a developer may make changes to the original, remote repository
+that may be critical to the success/functionality of one's local copy. But, this command
+does not yet alter any of one's own local branches.
+'git pull' brings changes in the remote repository into the the code/local branches and merges any changes that may have been made
+in the remote repository. 'git pull' actually performs a 'git fetch' before merging any remote changes that
```

Comment:
Yes, this is good! But just to be a clear, a git fetch will grab the changes but doesn't merge them :)